### PR TITLE
Updated Italian

### PR DESCRIPTION
**Pull request by Raffox97**
- Removed useless naming (Not made by me)
- Added the newly added Portugal islands

### CODE REVIEW

File: web/app/configs/lang.json [hunk 1]
```diff
@@ -535,8 +535,8 @@
             "EH": "Sahara Occidentale",
             "ER": "Eritrea",
             "ES": "Spagna",
-            "ES-CN": "Spagna (Isole Canarie)",
```
Comment (corradio) [inline web/app/configs/lang.json]: To be consistent I think it is better to prefix with the country (as we do in Canada for example)

File: web/app/configs/lang.json [hunk 2]
```diff
@@ -633,6 +633,8 @@
             "PR": "Porto Rico",
             "PS": "Palestina",
             "PT": "Portogallo",
+            "PT-AC": "Azzorre",
```
Comment (corradio) [inline web/app/configs/lang.json]: Same here

**Comment by Raffox97** [comment]
The fact Is that no country have that naming type.

**Comment by corradio** [comment]
Mmh I think there must be a misunderstanding.
I've followed that convention for English at least, see for example https://github.com/Raffox97/electricitymap/blob/f9c284c1cc75c618b8d5ebb5c6049a20faa2cb44/web/app/configs/lang.json#L172.
Do you think we should rethink it?
Maybe we should enforce consistency across all languages, as I haven't been very strict on those  things for now.

**Comment by corradio** [comment]
Hello @Raffox97 - any progress here?

**Comment by Raffox97** [comment]
I'm closing that since I will make a new PR later.

**Comment by Raffox97** [comment]
I've just created the PR #700

**Pull request closed without merging**